### PR TITLE
Throw errors in staging for broken links

### PR DESCRIPTION
**Pull request by cvalarida**
## Description
Like it says on the tin, if there are broken links, this should fail the staging build as well as production.

### Background
https://github.com/department-of-veterans-affairs/vets-website/pull/9884 removed this functionality, [but the team decided](https://dsva.slack.com/archives/CBU0KDSB1/p1563996823066900?thread_ts=1563920536.025000&cid=CBU0KDSB1) the cost now outweighs the benefit, so we're reverting.

## Testing done
@bderfer68 Who's the best person to ask about how to test this? I'm not super familiar with all the pieces just yet.

@rianfowler @gunsch for visibility.

## Screenshots
N/A

## Acceptance criteria
- [ ] The build passes
- [ ] Broken links in staging fail the build

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by ethanteague — APPROVED** [review]
LGTM

**Review by rianfowler — APPROVED** [review]

**Comment by rianfowler** [comment]
Some broken links are preventing this PR. maybe merge master? 

```

Error: You have 134 broken links:



outreach-and-events/outreach-materials/index.html >>> href: "/files/2019-05/What%20you%20need%20to%20know%20about%20VA%20Home%20Loan%20Guaranty_0.pdf", text: "Download PDF

                        (0.22MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/files/2019-05/Service%20dogs%20and%20other%20lesser-known%20VA%20benefits.pdf", text: "Download PDF

                        (0.20MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/files/2019-05/Lesser-known%20benefits%20to%20the%20Post%20911%20GI%20Bill.pdf", text: "Download PDF

                        (0.29MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/files/2019-05/Kick-start%20your%20new%20career%20with%20VA%20benefits.pdf", text: "Download PDF

                        (0.25MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/img/2019-06/VA-020%20Web%20Badges%20300x250%201.4.jpg", text: "Veterans: Find out how you can benefit. VA.gov"

outreach-and-events/outreach-materials/index.html >>> href: "/img/2019-06/VA-020%20Web%20Badges%20300x250%201.4.jpg", text: "Download JPG

                        (0.16MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/img/2019-06/VA-020%20Web%20Badges%20300x250%201.42.jpg", text: "Benefits for those who served. VA.gov"

outreach-and-events/outreach-materials/index.html >>> href: "/img/2019-06/VA-020%20Web%20Badges%20300x250%201.42.jpg", text: "Download JPG

                        (0.20MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/img/2019-06/VA-020%20Web%20Badges%201.5.jpg", text: "Veterans: Find out how you can benefit."

outreach-and-events/outreach-materials/index.html >>> href: "/img/2019-06/VA-020%20Web%20Badges%201.5.jpg", text: "Download JPG

                        (0.04MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/img/2019-06/VA-020%20Web%20Badges%201.52.jpg", text: "Benefits for those who served. VA.gov"

outreach-and-events/outreach-materials/index.html >>> href: "/img/2019-06/VA-020%20Web%20Badges%201.52.jpg", text: "Download JPG

                        (0.04MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/img/2019-06/VA-020%20Web%20Badges%201.53.jpg", text: "VA serves Veterans and their families. Explore VA benefits. VA.gov"

outreach-and-events/outreach-materials/index.html >>> href: "/img/2019-06/VA-020%20Web%20Badges%201.53.jpg", text: "Download JPG

                        (0.04MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/files/2019-06/VA-013%20Toolkit%20Burials%20and%20Memorials%20Fact%20Sheet%2002%20508.pdf", text: "Download PDF

                        (2.54MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/files/2019-06/VA-027%20Fact%20Sheet%20Demographics%20Disabled%20Vet%20508.pdf", text: "Download PDF

                        (2.48MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/files/2019-06/VA-027%20Fact%20Sheet%20Demographics%20Rural%20Vet%20508.pdf", text: "Download PDF

                        (2.46MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/files/2019-06/VA-027%20Fact%20Sheet%20Demographics%20Seniors%20508.pdf", text: "Download PDF

                        (2.48MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/files/2019-06/VA-027%20Fact%20Sheet%20Demographics%20Women%20508.pdf", text: "Download PDF

                        (2.67MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/files/2019-06/190111%20ChooseVA%20Marketing%20Flyer%202019.docx", text: "Download DOCX

                        (0.49MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/files/2019-06/EVA%20-%20Outreach%20Tips%20508.pdf", text: "Download PDF

                        (5.26MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/files/2019-06/EVA-019_VA_Explore_family%20member_1.5_508.pdf", text: "Download PDF

                        (1.62MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/files/2019-06/EVA-019_VA_Explore_home%20loans_1.5_508.pdf", text: "Download PDF

                        (1.38MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/files/2019-06/EVA-019_VA_Explore_life%20insurance_1.5_508.pdf", text: "Download PDF

                        (1.40MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/files/2019-06/VA-019_VA_Health%20Care%20female%201.0_508.pdf", text: "Download PDF

                        (7.84MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/files/2019-06/VA-019_VA_Memorial%20Benefits%201.0_508.pdf", text: "Download PDF

                        (10.46MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/img/2019-06/VA-020%20Web%20Badges%201.54.jpg", text: "VA serves Veterans and their families. Explore VA benefits. VA.gov"

outreach-and-events/outreach-materials/index.html >>> href: "/img/2019-06/VA-020%20Web%20Badges%201.54.jpg", text: "Download JPG

                        (0.04MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/files/2019-06/EVA-022%20Wallet%20Cards%208%20up%202.0%20508.pdf", text: "Download PDF

                        (1.98MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/img/2019-06/VA-020%20Web%20Badges%20170x155%201.0.jpg", text: "Veterans: Find out how you can benefit. VA.gov"

outreach-and-events/outreach-materials/index.html >>> href: "/img/2019-06/VA-020%20Web%20Badges%20170x155%201.0.jpg", text: "Download JPG

                        (0.07MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/img/2019-06/VA-020%20Web%20Badges%20180x150%201.42.jpg", text: "Benefits for those who served. VA.gov"

outreach-and-events/outreach-materials/index.html >>> href: "/img/2019-06/VA-020%20Web%20Badges%20180x150%201.42.jpg", text: "Download JPG

                        (0.09MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/img/2019-06/VA-020%20Web%20Badges%20170x155%201.02.jpg", text: "Benefits for those who served. VA.gov"

outreach-and-events/outreach-materials/index.html >>> href: "/img/2019-06/VA-020%20Web%20Badges%20170x155%201.02.jpg", text: "Download JPG

                        (0.09MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/img/2019-05/VA-020%20Web%20Badges%20300x250%201.4.jpg", text: "Veterans smiling"

outreach-and-events/outreach-materials/index.html >>> href: "/img/2019-05/VA-020%20Web%20Badges%20300x250%201.4.jpg", text: "Download JPG

                        (0.16MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/img/2019-06/VA-020%20Web%20Badges%20180x150%201.4.jpg", text: "Veterans: Find out how you can benefit. VA.gov"

outreach-and-events/outreach-materials/index.html >>> href: "/img/2019-06/VA-020%20Web%20Badges%20180x150%201.4.jpg", text: "Download JPG

                        (0.07MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/files/2019-06/VA-019_VA_Health%20Care%20male%201.0_508.pdf", text: "Download PDF

                        (10.43MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/img/2019-06/VA-020%20Web%20Badges%20184x90%20%28al%29%201.02.jpg", text: "Benefits for those who served. VA.gov"

outreach-and-events/outreach-materials/index.html >>> href: "/img/2019-06/VA-020%20Web%20Badges%20184x90%20%28al%29%201.02.jpg", text: "Download JPG

                        (0.06MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/files/2019-06/VA-017%20Brochure%20One%20Pager%203.2.pdf", text: "Download PDF

                        (3.54MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/files/2019-06/VA%20-%20Toolkit%209%20Careers%20and%20Employment%20Fact%20Sheet%20508.pdf", text: "Download PDF

                        (2.25MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/files/2019-06/VA%20-%20Toolkit%209%20Disability%20Fact%20Sheet%20508.pdf", text: "Download PDF

                        (2.48MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/img/2019-06/VA-020%20Web%20Badges%20184x90%20%28al%29%201.0.jpg", text: "Veterans: Find out how you can benefit. VA.gov"

outreach-and-events/outreach-materials/index.html >>> href: "/img/2019-06/VA-020%20Web%20Badges%20184x90%20%28al%29%201.0.jpg", text: "Download JPG

                        (0.05MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/img/2019-06/VA-020%20Web%20Badges%20205x175%20%28al%29%201.0.jpg", text: "Veterans: Find out how you can benefit. VA.gov"

outreach-and-events/outreach-materials/index.html >>> href: "/img/2019-06/VA-020%20Web%20Badges%20205x175%20%28al%29%201.0.jpg", text: "Download JPG

                        (0.09MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/img/2019-06/VA-020%20Web%20Badges%20205x175%20%28al%29%201.02.jpg", text: "Benefits for those who served. VA.gov"

outreach-and-events/outreach-materials/index.html >>> href: "/img/2019-06/VA-020%20Web%20Badges%20205x175%20%28al%29%201.02.jpg", text: "Download JPG

                        (0.11MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/files/2019-06/VA%20-%20Toolkit%209%20Family%20Member%20Fact%20Sheet%20508.pdf", text: "Download PDF

                        (2.37MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/files/2019-06/VA%20-%20Toolkit%209%20Housing%20Assistance%20Fact%20Sheet%20508_0.pdf", text: "Download PDF

                        (2.54MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/files/2019-06/VA%20-%20Toolkit%209%20Life%20Insurance%20Fact%20Sheet%20508.pdf", text: "Download PDF

                        (2.58MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/files/2019-06/VA%20-%20Toolkit%209%20PensionFact%20Sheet%20508.pdf", text: "Download PDF

                        (2.54MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/files/2019-05/After%20war%2C%20a%20Veteran%20learns%20to%20thrive.pdf", text: "Download PDF

                        (0.27MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/files/2019-05/Education%20and%20Training%20Fact%20Sheet%20508.pdf", text: "Download PDF

                        (3.70MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/files/2019-05/VA%20Disability%20Is%20it%20for%20you.pdf", text: "Download PDF

                        (0.18MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/files/2019-05/Health%20Care%20Fact%20Sheet%20508.pdf", text: "Download PDF

                        (2.27MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/files/2019-05/VA%20employment%20services%20poster.pdf", text: "Download PDF

                        (1.75MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/files/2019-05/VA%20life%20insurance%20poster.pdf", text: "Download PDF

                        (1.40MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/files/2019-05/VA%20benefits%20poster.pdf", text: "Download PDF

                        (3.54MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/files/2019-05/VA%20disability%20compensation%20poster.pdf", text: "Download PDF

                        (0.85MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/files/2019-05/VA%20education%20benefits%20poster.pdf", text: "Download PDF

                        (2.29MB)

                        "

outreach-and-events/outreach-materials/index.html >>> href: "/img/2019-05/Transition%20Infographic.jpg", text: "Image of two enlisted soldiers"

outreach-and-events/outreach-materials/index.html >>> href: "/img/2019-05/Transition%20Infographic.jpg", text: "Download JPG

                        (1.34MB)

                        "

health-care/pay-copay-bill/index.html >>> href: "/health-care/pay-copay-bill/dispute/", text: "Learn how to dispute your copay charges"

health-care/pay-copay-bill/index.html >>> href: "/health-care/pay-copay-bill/financial-hardship/", text: "Learn how to request financial hardship assistance"

health-care/pay-copay-bill/index.html >>> href: "/health-care/pay-copay-bill/dispute/", text: "

            Dispute your copay charges

        

            If you disagree with the charges or amounts on your bill, find out how to file a dispute.

    "

health-care/pay-copay-bill/index.html >>> href: "/health-care/pay-copay-bill/financial-hardship/", text: "

            Request financial hardship assistance

        

            If you're struggling to pay your copays, learn how to request a repayment plan, debt relief, or copay exemption.

    "

health-care/pay-copay-bill/index.html >>> href: "/health-care/about-va-health-benefits/health-care-costs/", text: "

            Your health care costs

        

            Learn how we assess and verify your income to help determine if you're eligible for VA health care and whether you'll need to pay copays for certain types of care, tests, and medications.

    "

health-care/copay-rates/index.html >>> href: "/health-care/about-va-health-benefits/health-care-costs/", text: "Learn how we determine whether you'll pay copays"

health-care/copay-rates/index.html >>> href: "/health-care/about-va-health-benefits/health-care-costs/", text: "Find out how we determine your health care costs"

health-care/copay-rates/index.html >>> href: "/health-care/about-va-health-benefits/health-care-costs/", text: "

            Your health care costs

        

            Learn how we assess and verify your income to help determine if you're eligible for VA health care and whether you'll need to pay copays for certain types of care, tests, and medications.

    "

pittsburgh-health-care/index.html >>> href: "/img/styles/crop_7_2/public/2019-04/UD%20front.jpg", text: "Pittsburgh VAMC University Drive Facility Image"

pittsburgh-health-care/index.html >>> href: "/pittsburgh-health-care/events/veterans-town-hall-0", text: "Veterans Town Hall"

pittsburgh-health-care/index.html >>> href: "", text: "Pittsburgh VA Medical Center-University Drive"

pittsburgh-health-care/index.html >>> href: "/pittsburgh-health-care/events/veterans-town-hall-1", text: "Veterans Town Hall"

pittsburgh-health-care/index.html >>> href: "", text: "H.J. Heinz III Campus"

pittsburgh-health-care/press-releases/index.html >>> href: "", text: "

          1

        "

pittsburgh-health-care/press-releases/page-2/index.html >>> href: "", text: "

          2

        "

pittsburgh-health-care/stories/index.html >>> href: "", text: "

          1

        "

pittsburgh-health-care/stories/page-2/index.html >>> href: "", text: "

          2

        "

pittsburgh-health-care/events/index.html >>> href: "", text: "H.J. Heinz III Campus"

pittsburgh-health-care/events/index.html >>> href: "/pittsburgh-health-care/events/women-veterans-tea", text: "Women Veterans Tea"

pittsburgh-health-care/events/index.html >>> href: "", text: "Pittsburgh VA Medical Center-University Drive"

pittsburgh-health-care/events/index.html >>> href: "", text: "Washington County outpatient clinic "

pittsburgh-health-care/events/index.html >>> href: "/pittsburgh-health-care/events/veterans-town-hall", text: "Veterans Town Hall"

pittsburgh-health-care/events/index.html >>> href: "", text: "H.J. Heinz III Campus"

pittsburgh-health-care/events/index.html >>> href: "/pittsburgh-health-care/events/veterans-town-hall-0", text: "Veterans Town Hall"

pittsburgh-health-care/events/index.html >>> href: "", text: "Pittsburgh VA Medical Center-University Drive"

pittsburgh-health-care/events/index.html >>> href: "/pittsburgh-health-care/events/veterans-town-hall-1", text: "Veterans Town Hall"

pittsburgh-health-care/events/index.html >>> href: "", text: "H.J. Heinz III Campus"

pittsburgh-health-care/leadership/index.html >>> href: "/img/styles/1_1_square_medium_thumbnail/public/2019-05/Barb%20Forsha_20190509_0001.jpg", text: "Barbara Forsha"

pittsburgh-health-care/leadership/index.html >>> href: "/img/styles/1_1_square_medium_thumbnail/public/2019-05/Dr.%20Ali%20Sonel_20190307_0001.jpg", text: "Dr. Ali Sonel"

pittsburgh-health-care/leadership/index.html >>> href: "/img/styles/1_1_square_medium_thumbnail/public/2019-05/Lovetta%20Ford%202.jpg", text: "Lovetta Ford"

pittsburgh-health-care/leadership/index.html >>> href: "/img/styles/1_1_square_medium_thumbnail/public/2019-05/Jason%20Fay_7798.jpg", text: "Jason Fay"

pittsburgh-health-care/leadership/index.html >>> href: "/img/styles/1_1_square_medium_thumbnail/public/2019-05/Nick%20Haller_20190322_0006.jpg", text: "Nicholas J. Haller"

pittsburgh-health-care/leadership/index.html >>> href: "/img/styles/1_1_square_medium_thumbnail/public/2019-05/Alan%20Petrazzi_20190308_0005_edit.jpg", text: "Alan Petrazzi"

health-care/pay-copy-bill/financial-hardship/index.html >>> href: "/health-care/about-va-health-benefits/health-care-costs/", text: "
```

**Comment by ethanteague** [comment]
@rianfowler @cvalarida - can we close this pull request (apologies)? We were going to use this to troubleshoot prod, but we've since resolved, and I'm not sure the rest of the team wants to implement this on stage.

**Comment by gunsch** [comment]
It seems like having it off in staging was one of the reasons we didn't detect this until prod, and that we wanted it turned back on in staging for better sanity going forward in general. Did that reasoning change, @ethanteague ?

**Comment by ethanteague** [comment]
> It seems like having it off in staging was one of the reasons we didn't detect this until prod, and that we wanted it turned back on in staging for better sanity going forward in general. Did that reasoning change, @ethanteague ?

This is mostly true. There was an immediate need to turn it on to better debug the situation we had on prod, but since we've resolved said issue, the need is no longer urgent. The concern that was raised by some of our team about reenabling this is that staging is often used for training and demonstration purposes, and having link-checker on presents another risk (we won't be able to demo / train when link-checker issues arise). So, I'm looping in @kevwalsh and @jefflbrauer for a little more discussion about this.

**Comment by batemapf** [comment]
adding @andyhawks and @bderfer68 to the convo, too. the ask at the 7/27 platform <--> CMS coordinating call was to go ahead with getting the link checker reenabled on staging.

**Comment by cvalarida** [comment]
I'm happy to do whatever, really.

My two cents: I'm less concerned about resolving a single instance of broken links breaking master builds and more concerned about preventing it from happening in the first place. If this will take a step in that direction, that's what I'd like to see, personally.

As far as the demos go, my gut says we can find another way around that, and the cost (pretty frequently breaking all the builds on all the branches for all the teams) outweighs the benefits. That said, I'm sure there's a truckload or two of context I don't have. :man_shrugging: 

Like I said, I'm happy to do whatever. I just want to move in a direction that stops breaking builds. Ounce of prevention vs pound of cure and all that.

**Comment by batemapf** [comment]
@cvalarida please merge this one in on or after 8/14. also, fwiw, [here](https://github.com/department-of-veterans-affairs/vets-website/pull/10527) is the work @ncksllvn did on the link checker itself.  cc @JeffBarnesUSDS

**Comment by ncksllvn** [comment]
This plugin was rewritten, btw, so the change will need to be made [here](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/stages/build/plugins/check-broken-links/index.js#L51) instead.

We may also want to hold off until this work is done, https://github.com/department-of-veterans-affairs/va.gov-team/issues/887. That ticket came out of our convo yesterday between Platform/Global UX/CMS and may really help crack down on the sneaky broken links. I labeled that with my team, but that could easily belong to Platform if you're interested.

File: src/site/stages/build/plugins/check-cms-urls.js
Comment (ncksllvn) [inline src/site/stages/build/plugins/check-cms-urls.js]: I'm not sure about this one. This ultimately makes sure that there aren't any URLs of the internal CMS `prod.cms.va.gov`, for example, in the Drupal content. They should have been updated to be the VA domain, or with the domain omitted. Maybe this should also fail on Staging? Up to you

**Review by ncksllvn — APPROVED** [review]

**Comment by cvalarida** [comment]
@ethanteague Hey! I'm looping back around to this per @batemapf's request.

We've got a good news / bad news sort of scenario here.

Good news: The PR successfully flags any broken links in staging.
Bad news: There are 206 broken links in staging so I can't merge it. :disappointed: 

What can we do to take care of that?

**Comment by ethanteague** [comment]
> @ethanteague Hey! I'm looping back around to this per @batemapf's request.
> 
> We've got a good news / bad news sort of scenario here.
> 
> Good news: The PR successfully flags any broken links in staging.
> Bad news: There are 206 broken links in staging so I can't merge it. 
> 
> What can we do to take care of that?

I dunno, @cvalarida.
Couple of things:

- Not sure if everybody on CMS team wants this on staging. 
- If everybody does, we will need to to create a task to fix these broken links

Punting to @kevwalsh @swirtSJW

**Comment by swirtSJW** [comment]
My understanding was that this was on hold per @ncksllvn 

> We may also want to hold off until this work is done, department-of-veterans-affairs/va.gov-team#887. 

@ethanteague  @cvalarida Sorry I can't see the error report as my keys have not been approved yet to give me SOCKS access to Jenkins so I need someone to read the link failures for me.  Were they systemic, or editorial in nature?

**Comment by ethanteague** [comment]
> My understanding was that this was on hold per @ncksllvn
> 
> > We may also want to hold off until this work is done, [department-of-veterans-affairs/va.gov-team#887](https://github.com/department-of-veterans-affairs/va.gov-team/issues/887).
> 
> @ethanteague @cvalarida Sorry I can't see the error report as my keys have not been approved yet to give me SOCKS access to Jenkins so I need someone to read the link failures for me. Were they systemic, or editorial in nature?

They're editorial, @swirtSJW

**Comment by kevwalsh** [comment]
@cvalarida @rianfowler @ncksllvn We're using staging for Migrations QA right now, as well as demo purposes for the Pittsburgh product, publishing PGH content that won't get published live for at least another sprint. Our devops team is working on various Unity tasks so that we can move those two processes away from staging. It sounds like it's pretty close, but they need another week or so to stabilize it. @ElijahLynn can speak better to the timeline.

**Comment by swirtSJW** [comment]
@JeffBarnesUSDS The use of Unity to replace Dev and Stage is almost there but it is not fully / reliably functional at the moment.   The Unity environments however are running the same link checker test that the VSP team is running so their should be fewer surprises.

It would still be great if https://github.com/department-of-veterans-affairs/va.gov-team/issues/887 were completed before link checking were made  build blocker as well as the pre-prod build happening hours before the prod deployment to give editors the chance to fix bad links before it becomes a prod deployment blocker.   Content currently is always in flux on staging so the link checker running as an alert on stage is helpful, the link checker blocking deployment of staging is not.

**Pull request closed without merging**
